manifests/v/VideoLAN/VLC/3.0.18/VideoLAN.VLC.installer.yaml

```
@@ -0,0 +1,96 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: VideoLAN.VLC
+PackageVersion: 3.0.18
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Commands:
+- vlc
+FileExtensions:
+- 264
+- 3ga
+- 3gp
+- aac
+- avi
+- cda
+- dash
+- dvr
+- flac
+- ifo
+- m2t
+- m2ts
+- m3u8
+- m4v
+- mkv
+- mov
+- mp3
+- mp4
+- mpg
+- mts
+- ogg
+- ogv
+- opus
+- pls
+- rec
+- rmvb
+- snd
+- sub
+- ts
+- vob
+- webm
+- wma
+- wmv
+- zab
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  InstallerUrl: http://get.videolan.org/vlc/3.0.18/win64/vlc-3.0.18-win64.msi
+  InstallerSha256: A618D66B2F753343402200AD17D92EB6D4AB10DB0DD1A098402E2968AA3114C4
+  ProductCode: '{B022B1C5-D067-42CB-98E2-D965E4D74CFE}'
+  AppsAndFeaturesEntries:
+  - DisplayName: VLC media player 3.0.18 (64-bit)

```

```suggestion
  - DisplayName: VLC media player
```
Must match Apps&Features/Control Panel.

---

Ahh, allright, I "reused" this from the previous manifest and simply adjusted the version number...